soes/esc_coe.h

```diff
@@ -13,8 +13,8 @@
 
 #include <cc.h>
 
-CC_PACKED_BEGIN
-typedef struct CC_PACKED
+
```

Comment:
Was this intentional? I don't think these structures actually need packing but there should be a note in the commit message explaining why it was removed.

Reply:
Yes, added it as a seperate commit. I beleive we should avoid force packing of structs. Let the compiler handle it.

Reply:
Agreed, if the struct does not represent a protocol packet.